service/servicelayerfunctionalities.cc

```diff
@@ -162,6 +162,40 @@ Status ServerForCommandLineClient::chirp(ServerContext *context,
     }
     client_key.put("reply" + request->parent_id(), reply_chirp_to_proto);
   }
+
+  /* Check for hashtag in chirp - if so add to KVS as a hashtag chirp */
```

Comment:
Maybe put this part in another function since chirp() is pretty long already?

Reply:
good idea